### PR TITLE
Bake conflicting composer dependencies into namespace

### PR DESCRIPTION
**Pull request by mcerva**
Baked credis, monolog and Symfony/Yaml into project namespace.

Close https://github.com/QualityUnit/resqu-server/issues/6

### CODE REVIEW

**Comment by jperdochqu** [comment]
@dmolnarqu @jzernovic Maybe will be fine to finish this before https://github.com/QualityUnit/infrastructure-issues/issues/736 start.

File: lib/Libs/Symfony/Yaml/Unescaper.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Resque\Libs\Symfony\Yaml;
+
+
+use Resque\Libs\Symfony\Yaml\Exception\ParseException;/**
```
Comment (jzernovic) [inline lib/Libs/Symfony/Yaml/Unescaper.php:15]: Formatting.

File: lib/Libs/Symfony/Yaml/Yaml.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Resque\Libs\Symfony\Yaml;
+
+
+use Resque\Libs\Symfony\Yaml\Exception\ParseException;/**
```
Comment (jzernovic) [inline lib/Libs/Symfony/Yaml/Yaml.php:15]: Formatting.

**Review by jzernovic — CHANGES_REQUESTED** [review]

File: lib/Libs/Symfony/Yaml/Inline.php [hunk 3]
```diff
@@ -0,0 +1,811 @@
...
+            case '!' === $scalar[0]:
+                switch (true) {
+                    case 0 === strpos($scalar, '!!str '):
+                        return (string) substr($scalar, 6);
+                    case 0 === strpos($scalar, '! '):
+                        return substr($scalar, 2);
+                    case 0 === strpos($scalar, '!php/object'):
+                        if (self::$objectSupport) {
+                            if (!isset($scalar[12])) {
+                                trigger_deprecation('symfony/yaml', '5.1', 'Using the !php/object tag without a value is deprecated.');
```
[the remark this answers is not in the document]
Comment (mcerva) [reply lib/Libs/Symfony/Yaml/Inline.php:606]: I added the missing function, the rest should not be a problem as it just calls @trigger_error

**Review by jzernovic — APPROVED** [review]

File: lib/Libs/Colinmollenhour/Credis/Client.php [hunk 4]
```diff
@@ -0,0 +1,1494 @@
...
+                        $this->close(true);
+                        $this->connect();
+                        $response = call_user_func_array(array($this->redis, $name), $args);
+                    } else {
+                        throw $e;
+                    }
+                }
+            }
+            // Wrap exceptions
+            catch(RedisException $e) {
```
Comment (dmolnarqu) [inline lib/Libs/Colinmollenhour/Credis/Client.php:1183]: class not imported

File: lib/Libs/Psr/Log/LoggerInterface.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+    /**
+     * Logs with an arbitrary level.
+     *
+     * @param mixed   $level
+     * @param string  $message
+     * @param mixed[] $context
+     *
+     * @return void
+     *
+     * @throws \Psr\Log\InvalidArgumentException
```
Comment (dmolnarqu) [inline lib/Libs/Psr/Log/LoggerInterface.php:122]: wrong class

File: lib/Libs/Symfony/Yaml/Inline.php [hunk 6]
```diff
@@ -0,0 +1,811 @@
...
+        switch (true) {
+            case \is_resource($value):
+                if (Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE & $flags) {
+                    throw new DumpException(sprintf('Unable to dump PHP resources in a YAML file ("%s").', get_resource_type($value)));
+                }
+
+                return self::dumpNull($flags);
+            case $value instanceof \DateTimeInterface:
+                return $value->format('c');
+            case $value instanceof \UnitEnum:
```
Comment (dmolnarqu) [inline lib/Libs/Symfony/Yaml/Inline.php:130]: UnitEnum is only available in PHP 8.1 - missing polyfill

File: lib/Libs/Monolog/bootstrap.php [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Resque\Libs\Monolog\Polyfill\Php80 as p;
+
+if (\PHP_VERSION_ID >= 80000) {
+    return;
+}
```
Comment (dmolnarqu) [inline lib/Libs/Monolog/bootstrap.php:16]: this file must be autoloaded for polyfill to work - make polyfill a separate namespace please

File: lib/Libs/Symfony/Yaml/function.php [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+if (!function_exists('trigger_deprecation')) {
```
Comment (dmolnarqu) [inline lib/Libs/Symfony/Yaml/function.php:12]: the file is never used - should it be autoloaded?

Comment (mcerva) [reply lib/Libs/Symfony/Yaml/function.php:12]: it is used in Inline.pp

File: lib/Libs/Symfony/Yaml/Inline.php (hunk 6 above)
Comment (mcerva) [reply lib/Libs/Symfony/Yaml/Inline.php:130]: should not be problem in older versions

File: lib/Libs/Symfony/Yaml/function.php (hunk 8 above)
Comment (dmolnarqu) [reply lib/Libs/Symfony/Yaml/function.php:12]: Used, yes, but is it loaded before being used?

**Comment by mcerva** [comment]
yes

**Comment by mcerva** [comment]
My bad I though bootstrap is autoloaded automaticly somehow... added it to composer

**Pull request merged**